contest/contest_archive.py

```diff
@@ -44,12 +44,19 @@ def get_owned_or_started_contests(user):
 def get_owned_contests(user):
     request = Q(owner = user)|Q(coowner = user)
     owned_contests = Contest.objects.order_by('-start_time').filter(request)
-    return owned_contests
+    return owned_contests.distinct()
 
 def get_started_contests():
     now = datetime.now()
     return Contest.objects.order_by('-start_time').filter(start_time__lte = now)
 
+def get_attended_contests(user):
```

Comment:
Something like this
https://github.com/bruce3557/NTHUOJ_web/blob/dev/contest/contest_info.py#L68